### ISSUE TITLE
Create referral program landing pages

### ISSUE THREAD

**Issue by Saltzshaker** [popuparchive/pop-up-archive#1320]
If an existing paid user gets gets someone to sign up for a paid plan, they'll both get 20% off the next month. Paid user sends out invites with promo codes. 

Emily will add wireframes to this ticket then assign to @AndersEverett to build out. https://github.com/popuparchive/pop-up-archive/issues/1334 is prereq.

**Comment by Saltzshaker** [comment]
![demo plan referral page](https://cloud.githubusercontent.com/assets/8497203/11731415/fff093be-9f4f-11e5-8592-93cec212d30d.png)
![paid plan referral page](https://cloud.githubusercontent.com/assets/8497203/11731414/fff03d56-9f4f-11e5-82e6-7e96d4ee7394.png)

**Comment by Saltzshaker** [comment]
Main goal is email verification, don't get too hung up on Gmail Contact thing. There might be an Angular email verification tool.

**Comment by Saltzshaker** [comment]
Hi,

I've been using Pop Up Archive — a simple web app that makes audio searchable with high-accuracy automatic transcripts — and thought you might be interested. Pop Up Archive saves me tons of time transcribing, and allows me to instantly search within audio files.

If you sign up with promo code #PopUpFriend, you can get 20% off of your first month on a paid plan. Click here to learn more and try it out!

Thanks,
{username}

**Comment by annewootton** [comment]
Looking good. Will Slack you edits.

**Comment by Saltzshaker** [comment]
*Revised email*
Subject line: Discount on audio transcription
Hi,

I've been using Pop Up Archive — a service that makes audio searchable with high-accuracy automatic transcripts — and thought you might be interested. Pop Up Archive saves me tons of time transcribing, and allows me to search for exact words and phrases within audio files.

If you sign up with promo code #PopUpFriend, you can get 20% off of your first month. There's also a free demo you can try out. Click here to learn more!

Thanks,
{username}

**Comment by AndersEverett** [comment]
@Saltzshaker On the 'click here' part of the revised email above, where should this link to?

**Comment by Saltzshaker** [comment]
The pricing page: popuparchive.com/pricing

**Comment by AndersEverett** [comment]
All three referral scenarios should be covered with 94189fd .

**Comment by Saltzshaker** [comment]
Ran through the referral flow and encountered a few issues. I'll summarize them with screenshots here. Then I'll talk to @AndersEverett and figure out the best tickets to make out of these issues.

**1. Demo users see Demo plan referral page variant** - YES - although they see combo message as page loads
![screen shot 2016-01-13 at 1 51 58 pm](https://cloud.githubusercontent.com/assets/8497203/12309595/617ead4e-ba00-11e5-8375-08464db9c613.png)

![referafriend](https://cloud.githubusercontent.com/assets/8497203/12309585/4f765f02-ba00-11e5-90b2-0bf2431da9eb.png)

Demo user sends 3 emails...
…Demo user can claim 20% discount on paid plan - YES
![demo-discount-upgrade](https://cloud.githubusercontent.com/assets/8497203/12309666/d77f6326-ba00-11e5-8b4a-6b96f1dd7092.png)

...People who receive emails can sign up with a 20% discount - NO. Discount text appears for signed in users, but the plan price is not discounted. The plan prices should be reduced, and the page should be visible even to new users who don’t yet have a Pop Up Archive account.
![popupfriend-discount](https://cloud.githubusercontent.com/assets/8497203/12309736/2feaaa98-ba01-11e5-8d9c-c6c1f2e09bb1.png)
![nodiscount-discount](https://cloud.githubusercontent.com/assets/8497203/12309741/3349f7c0-ba01-11e5-91a6-9e879bf03791.png)

**2.  Users on paid plans see paid plan referral page variant** - YES
![20for20](https://cloud.githubusercontent.com/assets/8497203/12309569/38b30b30-ba00-11e5-920c-f4d4282560be.png)

Paying user sends 3 emails...
- People who receive emails can sign up with a 20% discount - NO
- Paying user gets 20% discount when someone they refer signs up for a paid plan - ? When I try to upgrade in the UI using a Stripe test card (or real card), the subscribe modal gets stuck.
![screen shot 2016-01-13 at 11 58 59 am](https://cloud.githubusercontent.com/assets/8497203/12309840/a8689caa-ba01-11e5-81e9-df1accdeed92.png)

**Other:**
* Since we’re using urls, not “codes,” change mentions of the #DemoDiscount and #PopUpFriend in the email and text on /refer and /pricing pages. I’ll enter the replacement text in a new comment.
* also, with the urls, the promo code section on the bottom of the /refer page is no longer relevant, right?
![promo-code](https://cloud.githubusercontent.com/assets/8497203/12309815/84798c3c-ba01-11e5-8598-acf152aabb0f.png)
* @anders Should the emails be sent from anders@popuparchive.com?

**Comment by Saltzshaker** [comment]
View changes:
* Users invited to sign up should see this when they click on a plan: 
"Your #PopUpFriend discount will applied to your first invoice. Thanks!
Total: $ discounted plan amount" (following same model as above for #DemoDiscount
* Delete "Enter 3+ emails and press send to see your promo code!" Replace with "Make sure you're ready to redeem your discount as soon as you send the invites! You can only redeem your discount immediately after sending."

---

* Email

from: edison@popuparchive.com
subject: {Referrer's first name} sent you a discount for automatic transcription from Pop Up Archive
Hey there, 
 
Your friend {referrer's name} just invited you to join Pop Up Archive, a service that makes audio searchable with high-accuracy automatic transcripts. Pop Up Archive can help save you tons of time transcribing, and allows you to search for exact words and phrases within audio files. 
 
Click here to get 20% off of your first month with our #PopUpFriend discount: pop-up-archive.dev/pricing?ui=1048. There's also a free demo you can try out. Learn more at popuparchive.com/about.

Thanks! 
The Pop Up Archive Team

---

**Comment by Saltzshaker** [comment]
@AndersEverett will make the changes to the UI/mailers listed above. Plan changes aren't working in staging, so I'll test the rest of the functionality in production, before we start linking to the referral page. 

Also, @AndersEverett, is it possible to make the #PopUpFriend pricing page viewable to users who don't have accounts yet and thus aren't logged in?

**Comment by Saltzshaker** [comment]
Confirmed that new users trying to redeem the #PopUpFriend discount get discount messaging on the sign-up page. If possible, I think we should add UI messaging on the pricing page in this scenario as well so that it says "Get 20% off now with our #PopUpFriend discount!" Adding this change to the list of view changes above.
![screen shot 2016-01-14 at 3 28 41 pm](https://cloud.githubusercontent.com/assets/8497203/12341049/cb0b2e6c-bad3-11e5-82c3-07f302282fcd.png)

**Comment by Saltzshaker** [comment]
Moved UI changes to a new ticket in Megan https://github.com/popuparchive/pop-up-archive/issues/1355 and marking this as ready to deploy
